testing/src/main/java/org/oppia/android/testing/TextInputAction.kt

```diff
@@ -0,0 +1,40 @@
+package org.oppia.android.testing
+
+import android.content.Context
+import android.view.View
+import androidx.annotation.StringRes
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+import javax.inject.Inject
+
+class TextInputAction @Inject constructor(
+  val context: Context
+) {
+
+  fun hasErrorText(@StringRes expectedErrorTextId: Int): Matcher<View> {
+    return object : TypeSafeMatcher<View>() {
+      override fun matchesSafely(view: View): Boolean {
+        val expectedErrorText = context.resources.getString(expectedErrorTextId)
+        return (view as TextInputLayout).error == expectedErrorText
+      }
+
+      override fun describeTo(description: Description) {
+        description.appendText("TextInputLayout's error")
+      }
+    }
+  }
+
+  fun hasNoErrorText(): Matcher<View> {
+    return object : TypeSafeMatcher<View>() {
+      override fun matchesSafely(view: View): Boolean {
+        return (view as TextInputLayout).error.isNullOrEmpty()
+      }
+
+      override fun describeTo(description: Description) {
+        description.appendText("")
```

Comment:
Could you please update this line with

```kotlin
description.appendText("TextInputLayout's error")
```

Reply:
done

Reply:
Please say done after pushing the code, as for this pr this doesn't seems like done. Helpful to review.

Reply:
@Akshat-gour

Reply:
done